docs/output-filters.md

```diff
@@ -0,0 +1,36 @@
+Output Filters
```

Comment:
Thanks for writing this ... I have two thoughts. I don't know how to handle them though.

1. Output filters sounds a lot like output formats which is a related but different concept. Everyone is going to mix them up.
1. Might be worth saying how this feature differs from grep and when to use each.